codegen/test/test_parser.rb

```diff
@@ -44,22 +44,26 @@ def test_parse_invalid_method
 
   def test_parse_method_discardable_result
     parser = Parser.new(path: '', string: '
+      // This is a sample file
       TW_EXTERN_C_BEGIN
       struct TWEthereumAbiFunction;
 
+      // Ethereuem ABI helpers
       TW_EXPORT_CLASS
       struct TWEthereumAbiEncoder;
-      /// Encode function to Eth ABI binary
```

Comment:
should be `///`?

Reply:
?

Reply:
Yes, all lines starting with `//` (2 slashes) before an entity is being copied over to the generated source file as well.  So yes, this includes lines starting with 3 (or more) slashes as well.